Overview.src.html

```diff
@@ -3842,7 +3842,8 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 <h3>Body mixin</h3>
 
 <pre class=idl>typedef any <dfn>JSON</dfn>;
-typedef (<span data-anolis-spec=fileapi>Blob</span> or BufferSource or <span data-anolis-spec=xhr>FormData</span> or <span data-anolis-spec=url>URLSearchParams</span> or USVString) <dfn>BodyInit</dfn>;</pre>
+typedef (<span data-anolis-spec=fileapi>Blob</span> or BufferSource or <span data-anolis-spec=xhr>FormData</span> or <span data-anolis-spec=url>URLSearchParams</span> or USVString) <dfn>BodyInit</dfn>;
+typedef (<span>BodyInit</span> or <code title=concept-ReadableStream>ReadableStream</code>) <dfn>ResponseBodyInit</dfn>;</pre>
```

Comment:
I was about to merge, but you need to use `<span>` inside `<pre>`, not `<code>`. Otherwise the coloring gets weird. A `<pre class=idl>` block is code already in a way.